faq.md

```diff
@@ -135,6 +135,14 @@ Setting the default version affects a number of elements including (but not limi
 4. In the files tab, the default version's files would be displayed first.
 5. The docker pull command reflects the default version
 
+## How to add other users as maintainers of a tool?
```

Comment:
"How do I add other users as maintainers or a tool or workflow?" 
Similarly, "are lack" doesn't sound quite right.

Reply:
removing the "are lack" typo, unclear what to do about the first one.

Reply:
nvm